controllers/suite_test.go

```diff
@@ -62,9 +61,6 @@ var _ = BeforeSuite(func() {
 	err = infrav1.AddToScheme(scheme.Scheme)
 	Expect(err).NotTo(HaveOccurred())
 
-	err = infrav1.AddToScheme(scheme.Scheme)
-	Expect(err).NotTo(HaveOccurred())
```

Comment:
why is this being removed?

Reply:
Please look at the diff again. This is removed because it's duplicate code; the same lines appear just above.